### PR TITLE
docs: Add comments to vtex.env

### PR DESCRIPTION
**Pull request by tlgimenes**
## What's the purpose of this pull request?
This PR adds some comments to `vtex.env` file so people forking this repo understand better what they mean

### CODE REVIEW

File: vtex.env [hunk 1]
```diff
@@ -2,6 +2,9 @@
 NEXT_TELEMETRY_DISABLED=true
 
 # configures WebOps
-USE_NODE_MODULES_CACHE=true
-USE_FRAMEWORK_CACHE=true
+## Enable re-using node_modules from previous runs for faster builds
+USE_BUILD_CACHE=true
```
Comment (filipewl) [inline vtex.env:6]: Is `USE_BUILD_CACHE` just a new name for `USE_NODE_MODULES_CACHE`?

File: vtex.env [hunk 2]
```diff
@@ -2,6 +2,9 @@
 NEXT_TELEMETRY_DISABLED=true
 
 # configures WebOps
-USE_NODE_MODULES_CACHE=true
-USE_FRAMEWORK_CACHE=true
```
Comment (filipewl) [inline vtex.env:6]: Is `USE_FRAMEWORK_CACHE` not going to be used anymore or will it be covered by the new `USE_BUILD_CACHE`?

**Review by filipewl — APPROVED** [review]
Looks good to me! Just curious about some changes.

File: vtex.env (hunk 2 above)
Comment (tlgimenes) [reply vtex.env:6]:
```USE_BUILD_CACHE = USE_FRAMEWORK_CACHE && USE_NODE_MODULES_CACHE
```

USE_FRAMEWORK_CACHE and USE_NODE_MODULES_CACHE don't exist anymore

**Pull request merged**